### DISCUSSION TITLE
希望大模型翻译能支持调整参数

### DISCUSSION THREAD

**Discussion by Z-y-Liu** [immersive-translate/immersive-translate#2132] [Ideas]
### 新功能 | New Feature

希望插件里能支持用户自定义修改大模型的配置参数，如红框所示。
![image](https://github.com/user-attachments/assets/ef52c8e4-be96-46c9-ac16-c0dbc3c648ce)

### 为什么您想要添加这些功能 | Why you want to add these features

我在阿里云百炼里试用大模型翻译，发现调整参数能有效改善翻译效果。

**Comment by Z-y-Liu** [comment]
我注意到插件已可以调整Temperature这个参数，但改变它并没有影响翻译效果。所以还是希望能支持top_p的设置。

**Reply by keyboard3** [reply]
1.9.7 可以这么加

![QQ_1727172375140](https://github.com/user-attachments/assets/ac9648dc-baf3-4e2b-9e97-bdc3b0c9d0a9)
